hail/python/hailtop/batch/job.py

```diff
@@ -89,8 +89,8 @@ def __init__(self,
         self._job_id: Optional[int] = None
 
         self._inputs: Set[_resource.Resource] = set()
-        self._internal_outputs: Set[_resource.Resource] = set()
-        self._external_outputs: Set[_resource.Resource] = set()
+        self._internal_outputs: Set[_resource.ResourceFile] = set()
```

Comment:
Are you sure this can't be a ResourceGroup or a PythonResult?

Reply:
Yeah, you're right, this can also be a PythonResult, but AFAICT it is never a group. Groups add their constituent files to this variable.

Reply:
Fixed.